packages/blue-migration-sdk-viem/src/positions/borrow/blue.borrow.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  type ChainId,
+  DEFAULT_SLIPPAGE_TOLERANCE,
+  type Market,
+  type MarketId,
+  MathLib,
+  type Position,
+  getChainAddresses,
+} from "@morpho-org/blue-sdk";
+import type { BlueInputBundlerOperations } from "@morpho-org/bundler-sdk-viem";
+import { maxUint256 } from "viem";
+
+export namespace MigratableBorrowPosition_Blue {
+  export interface Args {
+    marketTo: MarketId;
+    collateralAmount: bigint;
+    borrowAmount: bigint;
+    slippageFrom?: bigint;
+    slippageTo?: bigint;
+  }
+}
+
+export interface MigratableBorrowPositionConfig_Blue {
+  chainId: ChainId;
+  market: Market;
+  position: Pick<Position, "borrowShares" | "user" | "collateral">;
+}
+
+export class MigratableBorrowPosition_Blue
+  implements MigratableBorrowPositionConfig_Blue
+{
+  public readonly market: Market;
+  public readonly position: Pick<
+    Position,
+    "borrowShares" | "user" | "collateral"
+  >;
+  public readonly chainId: ChainId;
+
+  constructor(config: MigratableBorrowPositionConfig_Blue) {
+    this.market = config.market;
+    this.position = config.position;
+    this.chainId = config.chainId;
```

Comment:
Is the `chainId` relevant this early? We don't store it for other entities of SDKs, because we don't care where or how the position was fetched
Best would be to require it in the function(s) that need(s) it